components/hubspot/sources/contact-updated/contact-updated.js

```diff
@@ -0,0 +1,58 @@
+const hubspot = require("../../hubspot.app.js");
+
+module.exports = {
+  key: "hubspot-contact-updated",
+  name: "Contact Updated",
+  description: "Emits an event for each time a contact is updated.",
+  version: "0.0.1",
+  props: {
+    hubspot,
+    db: "$.service.db",
+    timer: {
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: 60 * 15,
+      },
+    },
+  },
+
+  async run(event) {
+    const lastRun = this.db.get("updatedAfter") || this.hubspot.monthAgo();
+    const updatedAfter = new Date(lastRun);
+    const data = {
+      limit: 10,
```

Comment:
Here and in other places - are these limits maximums, or can you fetch more than 10 results at a time? Just thinking of ways to reduce API calls as you paginate.

Reply:
These limits can be whatever we want. I just set them as the default values, but let me know if I should increase these.

Reply:
I think you should increase to at least `100` to reduce the need to make multiple API calls for larger customers. That'll reduce the total component runtime.